components/ContactInfoBcros.vue

```diff
@@ -42,6 +42,22 @@
             <br />
             Monday to Friday, 8:30am - 4:30pm Pacific time
           </p>
+
+          <p class="mt-6 mb-0 wide-16">
+            For account setup and login support, contact us at:
+          </p>
+          <ul class="contact-info-list mt-0">
+            <li>
+              <span>Toll Free:</span>
+              <a href="tel:+1-877-370-1033">1-877-370-1033</a>
+            </li>
+          </ul>
+
+          <p class="mt-5 mb-0">
+            <strong>Hours of Operation:</strong>
+            <br />
+            Monday to Friday, 8:30am - 4:30pm Pacific time
```

Comment:
Please check with UI team whether "time" should be capitalized as in the design.

Also on line 43 above.

Reply:
I believe you are right. It is capitalized as "Time" in the design.

Reply:
Did you check with them? We don't capitalize "time" in other UIs. The design might be wrong.

Reply:
Actually we do have "Time" in the [auth-web UI](https://dev.bcregistry.ca/business/auth/home/decide-business). I'll check with Yui.

Reply:
I just received confirmation from Yui that we should use lowercase "time". It's now reverted to lowercase "time".